### ISSUE TITLE
predict on image with custom trained model

### ISSUE THREAD

**Issue by ck-amrahd** [WongKinYiu/YOLO#71]
**Issue Description**
I have trained the model on the custom dataset, tried to do inference using the inference notebook, created a model using create_model, passed the class_num to be my custom number, and tried to load the trained model using load state dict and torch load, but it's not working as expected. Do you have any examples of this?

**Additional Context**
Add any other context or screenshots about the issue here.

**Future Considerations**
Please suggest any potential future improvements related to this issue.

**Comment by yjmm10** [comment]
Did you have the value of AP during the validation  phase of training. 

I got the value that approach 0

**Comment by ck-amrahd** [comment]
Nope, I didn't get anything. It saved the trained weights, that's all.
